decidim-pages/config/locales/ca.yml

```diff
@@ -0,0 +1,5 @@
+---
+ca:
+  components:
```

Comment:
Nothing happens if I include the translations here. It actually gets imported into crowdin to be verified later.